refactoring.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Math.random();
+  }
+}
+
+class Collection {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.push(book);
+    localStorage.setItem('collection', JSON.stringify(this.data));
+    addToDisplay(book);
+  }
+
+  removeBook(id) {
+    const book = document.getElementById(id);
+    book.remove();
+    this.data = this.data.filter((bookObject) => bookObject.id !== id);
+    localStorage.setItem('collection', JSON.stringify(this.data));
+  }
+}
```

Comment:
- [ ] When we reload the page then books don't show which are in the localStorage means your application is not getting data from localStorage. :cry:

### **Quick Fix** :relaxed:
- First in the constructor you need to get the localStorage
- Second, you need to create a method to render your books on page